tools/gromacs/trj.xml

```diff
@@ -140,23 +154,27 @@
                 </conditional>
             </when>
             <when value="trjcat">
-                <param name="trj_input" type="data" format='xtc,trr' label="Input trajectories" help="In XTC or TRR format, but please do not mix the two. Please note this tool does not currently take order into account when concatenating." multiple="true"/>
+                <param name="trj_input" type="data" format="xtc,trr" label="Input trajectories" help="In XTC or TRR format, but please do not mix the two. Plese note that the order of concatation is the same as the consecutive order of these selected inputs." multiple="true"/>
+                <repeat name="inputtrajs" title="Select input trajectories" min="0">
+                    <param name="trj_input" type="data" format="xtc,trr" label="Input trajectories" help="In XTC or TRR format, but please do not mix the two. Plese note that the order of concatation is the same as the consecutive order of these selected inputs." multiple="true"/>
```

Comment:
```suggestion
                    <param name="trj_input" type="data" format="xtc,trr" label="Input trajectories" help="In XTC or TRR format, but please do not mix the two. Plese note that the order of concatation is the same as the consecutive order of these selected inputs." />
```

Reply:
Not very enthusiastic about the repeat here unless it is absolutely necessary. What if you want to concatenate a hundred trajectories?

Reply:
Can't you use a collection as input?

Reply:
there wasn't a clear way to do this. also, the collection automatically orders items based on history number, which may not always coincide with the order of the trajectories.

Reply:
This does not require a user to forced to build a collection for the sole purpose of just ensuring ordering in this tool. Also this allows the ordered concatenation of multiple collections and/or individual trajectories.

Reply:
Alright. @thepineapplepirate can you please remove the original input, so there is just a single param within a repeat with `multiple="true"`? Thanks!

Reply:
> Alright. @thepineapplepirate can you please remove the original input, so there is just a single param within a repeat with `multiple="true"`? Thanks!

that way both ways should be possible, similar to https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_cat/0.1.1